### DISCUSSION TITLE
Add Preview Props in HTML while exporting with export command

### DISCUSSION THREAD

**Discussion by ahmedivy** [resend/react-email#1569] [Ideas]
### Goals

When we run email export, the generated HTML contains nothing in place of the dynamic content. It would be great if these placeholders were populated with preview props.

This way, I can generate HTML that I can send using other languages as well. For example, in the case of a password reset email, I can add a preview prop as this

```ts
PasswordResetEmail.PreviewProps = {
  resetCode: "{resetCode}",
} as PasswordResetProps;
```

If the preview props populate correctly ("{resetCode}" in place of resetCode), I can then read the HTML file in Python and send in this way: 
```py
html.format(resetCode="123") 
```

**Comment by gabrielmfern** [comment]
You don't need PreviewProps to do that, PreviewProps is meant only for the preview server.

What you can do instead is something like:

```jsx
function Email({ resetCode = "{resetCode}" }) {
  return ...
}
```
